docs/docs/sql/float.md

```diff
@@ -2,15 +2,19 @@
 
 We support standard IEEE 754 floating point types.
 
-`double` is a 64-bit standard FP value.
+`double` is a 64-bit standard FP value.  Accepted synonyms are
+`float8` and `float64`.
```

Comment:
Floats aren’t ever specified by their byte length

Reply:
They are in SQL

Reply:
`float4` is so I can write Postgres tests... https://github.com/postgres/postgres/blob/master/src/test/regress/expected/float4.out

Reply:
Funky, if customers ever want `f16` or `f128` that could be problematic